cli/azd/cmd/env.go

```diff
@@ -514,36 +514,40 @@ func (eg *envGetValuesFlags) Bind(local *pflag.FlagSet, global *internal.GlobalC
 }
 
 type envGetValuesAction struct {
-	azdCtx    *azdcontext.AzdContext
-	console   input.Console
-	env       *environment.Environment
-	formatter output.Formatter
-	writer    io.Writer
-	flags     *envGetValuesFlags
+	azdCtx     *azdcontext.AzdContext
+	console    input.Console
+	envManager environment.Manager
+	formatter  output.Formatter
+	writer     io.Writer
+	flags      *envGetValuesFlags
 }
 
 func newEnvGetValuesAction(
 	azdCtx *azdcontext.AzdContext,
-	env *environment.Environment,
+	envManager environment.Manager,
 	console input.Console,
 	formatter output.Formatter,
 	writer io.Writer,
 	flags *envGetValuesFlags,
 ) actions.Action {
 	return &envGetValuesAction{
-		azdCtx:    azdCtx,
-		console:   console,
-		env:       env,
-		formatter: formatter,
-		writer:    writer,
-		flags:     flags,
+		azdCtx:     azdCtx,
+		console:    console,
+		envManager: envManager,
+		formatter:  formatter,
+		writer:     writer,
+		flags:      flags,
 	}
 }
 
 func (eg *envGetValuesAction) Run(ctx context.Context) (*actions.ActionResult, error) {
-	err := eg.formatter.Format(eg.env.Dotenv(), eg.writer, nil)
```

Comment:
I think this is missing, right?

it should be in the lines below, below the `return nil nil`